engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -268,24 +268,19 @@ class WorkflowActor(val workflowId: WorkflowId,
   }
 
   onTransition {
-    case oldState -> terminalState if terminalState.terminal =>
+    case (oldState, terminalState: WorkflowActorTerminalState) =>
```

Comment:
oh god, you shouldn't have said anything. you know you said that to the case block tyrant .... :'(